homeassistant/components/hue/sensor_base.py

```diff
@@ -136,17 +135,17 @@ def async_update_items(self):
             if sensor_config is None:
                 continue
 
-            base_name = api[item_id].name
-            primary_sensor = primary_sensor_devices.get(_device_id(api[item_id]))
-            if primary_sensor is not None:
-                base_name = primary_sensor.name
-            name = sensor_config["name_format"].format(base_name)
-
-            current[uniqueid] = sensor_config["class"](
-                api[item_id], name, self.bridge, primary_sensor=primary_sensor
-            )
+            for sensor in sensor_config:
+                base_name = api[item_id].name
```

Comment:
Looks like we can shorten what we include in the iteration block. Just include the lines where we use `sensor`. The other lines aren't affected.